From 06610fac0df75918230bb2b66bcd7eff6f8cc1d5 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 08 Mar 2022 07:23:57 +0200
Subject: [PATCH] [SW-52631] gaudi2: remove legacy versioning registers

The versioning information of the preboot and efuse is passed
to other FW components via the bootinfo data, and to the host driver
via the COMMS protocol.
The registers that were used for the legacy method in gaudi2 are
being removed.

Change-Id: I28e2f123021b99e9aebe4ea41112b23da3db666b
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 7c9286c..c710df4 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -181,8 +181,8 @@
 	regs->fw_upd_sts = mmUPD_STS;
 	regs->fw_upd_cmd = mmUPD_CMD;
 	regs->fw_upd_pending_sts = mmUPD_PENDING_STS;
-	regs->fuse_ver_offset = mmFUSE_VER_OFFSET;
-	regs->preboot_ver_offset = mmPREBOOT_VER_OFFSET;
+	regs->fuse_ver_offset = 0;	/* unused */
+	regs->preboot_ver_offset = 0;	/* unused */
 	regs->uboot_ver_offset = mmUBOOT_VER_OFFSET;
 	regs->hw_state = mmHW_STATE;
 	regs->kmd_msg_to_cpu = mmPSOC_GLOBAL_CONF_KMD_MSG_TO_CPU;
@@ -1300,8 +1300,8 @@
 	global_conf.upd_pending_sts_map = mmUPD_PENDING_STS;
 	global_conf.upd_cmd_map = mmUPD_CMD;
 	global_conf.upd_sts_map = mmUPD_STS;
-	global_conf.preboot_ver_off_map = mmPREBOOT_VER_OFFSET;
-	global_conf.mmfuse_ver_off_map = mmFUSE_VER_OFFSET;
+	global_conf.preboot_ver_off_map = 0;	/* unused */
+	global_conf.mmfuse_ver_off_map = 0;	/* unused */
 }
 
 void device_init_ca53_cfg(void)
